certora/specs/misc/EVC_Prop25_ResetsTransientStorage.spec

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
rename this file not to refer to `Prop25`. it seems like it's `CER-27`?

Reply:
Right, this seems to be part of https://linear.app/euler-labs/issue/CER-38/execution-context-restored (which is a child of CER-27). I just merged the contents of this file into the other spec file for CER-38 and now we can see that all the parts of that spec are covered.